.travis.yml

```diff
@@ -7,31 +7,38 @@ cache:
     - $HOME/.composer/cache/files
     - $HOME/symfony-bridge/.phpunit
 
-matrix:
-  fast_finish: true
-  include:
-    - php: 7.1
-    - php: 7.2
+php:
+  - 7.1
+  - 7.2
 
 env:
   global:
-    - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
-    - SYMFONY_PHPUNIT_VERSION=5.7
+    - SYMFONY_PHPUNIT_VERSION=6.5
```

Comment:
This one is now useless, phpunit-bridge will do the job alone =) It's just in case we need to change the default PHPUnit version. With legacy for example.

Reply:
Sure!
```
$ ./vendor/bin/simple-phpunit
PHPUnit 5.7.27 by Sebastian Bergmann and contributors.
```

Update reverted :/

Reply:
Depends on your php version / if you already installed it or not =)

Reply:
it's from Travis stack trace, using PHP 7.1.

But using PHP 7.2, version 6.5 of PHPUnit was installed.

It's safer to configure the version to install :)